src/constants/information.tsx

```diff
@@ -0,0 +1,93 @@
+import { Route } from '../routers/route';
+import {
+  AddProductIcon,
+  IColors,
+  MyFavoritesIcon,
+  MyProductsIcon,
+  MyPurchasesIcon,
+  MySalesIcon,
+  MyTemplatesIcon,
+  PrivateMessagesIcon,
+  PublicMessagesIcon,
+  ReviewsIcon,
+  UserIcon,
+} from '../components/shared/Icons/dashBoardIcons.tsx';
+import { ReactNode } from 'react';
+
+export const information = [
+  { id: 1, title: 'Про нас', route: Route.about },
+  { id: 2, title: 'Правила та умови', route: Route.rules },
+  { id: 3, title: 'Доставка', route: Route.delivery },
+  { id: 4, title: 'Оплата', route: Route.payment },
+];
+
+interface IMenu {
+  id: number;
+  title: string;
+  route: string;
+  // eslint-disable-next-line no-unused-vars
```

Comment:
also remove here and in another places (if you set it correctly in eslint settings file.

Reply:
it is just for interface and enum
if you know how to set no-unused-vars for interface and enum, assist me